queue_services/entity-emailer/requirements.txt

```diff
@@ -78,6 +78,6 @@ webcolors==1.13
 Werkzeug==1.0.1
 yarl==1.8.2
 zipp==3.15.0
-git+https://github.com/bcgov/business-schemas.git@2.18.14#egg=registry_schemas
+git+https://github.com/bcgov/business-schemas.git@2.18.17#egg=registry_schemas
```

Comment:
Updated the version to include the updated schema for extensions.

Reply:
We'll need to update the schema version here too.

https://github.com/bcgov/lear/blob/634129ec203e003091f496b3fa4efe1e29e345d2/queue_services/entity-emailer/requirements/bcregistry-libraries.txt#L1